distributions/standalone/assembly.xml

```diff
@@ -16,6 +16,13 @@
                 <exclude>lib/boot/exec/**</exclude>
             </excludes>
         </fileSet>
+        <fileSet>
```

Comment:
why do you need this? there is already a dependency set for services, around line 170